Index page for User Guide visualization topic. (#780)
o3de/o3de.org: content/docs/user-guide/visualization/_index.md

---
linkTitle: "Visualization"

> title: "Visualization systems in Open 3D Engine"

Reviewer: [Topic headings style](https://o3de.org/docs/contributing/to-docs/style-guide/format/#topic-headings)
```suggestion
title: "Visualization Systems in Open 3D Engine"
```

description: Learn about the various systems that provide animation and rendering in Open 3D Engine (O3DE).
weight: 1000
---

In Open 3D Engine (O3DE), you can bring actors to life with complex animated behaviors, create engaging cinematic sequences, and visualize your creations in stunning realism with O3DE's physically based renderer Atom. The topics in this section cover the concepts, tools, and processes used to achieve your creative vision.

| Topic | Description |
| - | - |

> | [Animation Editor](./animation) | With Animation Editor, you can visually construct complex animation behaviors and transitions using anim graphs. Behaviors can be driven and blended with events and states. Animation Editor also has tools you can use to create simulated objects and cloth colliders so that objects and clothing dynamically react to actor animation. |

Reviewer 2: - Use "**Animation Editor**", because that is its proper name. 
- For "anim graphs": If you are talking about the tool, **Anim Graph**, then capitalize and bold the term. If you are talking about the product asset of the Anim Graph tool, animation graph, use the full term "animation graph". (For subsequent references, you can use it's shortened term, "anim graph".)
```suggestion
| [Animation Editor](./animation) | With **Animation Editor**, you can visually construct complex animation behaviors and transitions using animation graphs. Behaviors can be driven and blended with events and states. Animation Editor also has tools you can use to create simulated objects and cloth colliders so that objects and clothing dynamically react to actor animation. |
```

| [Cameras](./cameras) | Cameras define the view to be rendered. You can use multiple cameras, switch between cameras, animate cameras through script and keyframes, and create complex camera and lighting rigs that respond to runtime events and simulations. |

> | [Cinematics](./cinematics) | With Track View, you can create cinematics or entire movies rendered in real time with complete control over multiple cameras. You can also add scripted sequences with behaviors, animations, effects, and more, that are camera independent and triggered by events such as input or pre-defined states. |

Reviewer 2: ```suggestion
| [Cinematics](./cinematics) | With **Track View**, you can create cinematics or entire movies rendered in real time with complete control over multiple cameras. You can also add scripted sequences with behaviors, animations, effects, and more, that are camera independent and triggered by events such as input or predefined states. |
```

| [Environments](./environments) | O3DE provides tools you can use to create dense, vibrant environments including tools for procedurally populating landscapes with dynamic vegetation that responds to actors and other dynamic entities. |

> | [Rendering](./rendering) | Atom provides O3DE with a high performance, cross-platform, physically based renderer. Atom also includes tools you can use to add lights, process images, create materials, and add post process effects. With AZSL, Atom's shader language, you have fine control of the rendering process. |

Reviewer 2: Use **"Atom Renderer"**, bold, because that is its proper name. (Use just "Atom" for subsequent references.)
```suggestion
| [Rendering](./rendering) | **Atom Renderer** provides O3DE with a high performance, cross-platform, physically based renderer. Atom also includes tools you can use to add lights, process images, create materials, and add post process effects. With AZSL, Atom's shader language, you have fine control of the rendering process. |
```
